incubator/foundry-vtt/templates/deployment.yaml

```diff
@@ -31,7 +33,7 @@ spec:
       containers:
       - name: {{ .Chart.Name }}
         securityContext:
-          {{- toYaml .Values.securityContext | nindent 12 }}
+          {{- toYaml (merge (dict  "runAsGroup" (default 1000 .Values.container.gid)  "runAsUser" (default 1000 .Values.container.uid)) .Values.securityContext) | nindent 12 }}
```

Comment:
without this, v12/13 of felddy's container fail to deploy due to FS permissions.  This just make everyone's life easier by setting it to the same UID/GID as the container expects with an option to override.